Formula/bazarr.rb

```diff
@@ -0,0 +1,95 @@
+require "language/node"
+
+class Bazarr < Formula
+  include Language::Python::Virtualenv
+
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  license "GPL-3.0-only"
```

Comment:
The license is `GPL-3.0-or-later`:

> This program is free software: you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation, either version 3 of the License, or
  (at your option) any later version.

Reply:
Yep